### PR TITLE
Fix IHADSS Baro Alt display and FMC Functions

### PR DESCRIPTION
**Pull request by BradMick**

### CODE REVIEW

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageFLTDraw.sqf [hunk 1]
```diff
@@ -24,6 +24,9 @@ private _airspeed = vectorMagnitude (velocity _heli vectorDiff wind);
 private _baroAlt  = getPosASL _heli # 2 * SCALE_METERS_FEET;
 private _radarAlt = getPos _heli # 2 * SCALE_METERS_FEET;
 _heli setUserMFDText [MFD_INDEX_OFFSET(MFD_TEXT_IND_FLT_BALT), (round (_baroAlt / 10) * 10) toFixed 0];
+if (_radarAlt > 50) then {
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageFLTDraw.sqf:27]: If the logic for formatting the radar altitude is identical across the MPD FLT Page and IHADSS Symbology, could we share it?

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAltitudeHoldEnable.sqf [hunk 2]
```diff
@@ -5,7 +5,10 @@ if (_heli getVariable "fza_ah64_altHoldActive" == false) then {
     private _curVel    = vectorMagnitude [velocityModelSpace _heli # 0, velocityModelSpace _heli # 1];
     private _curVelZ   = velocity _heli # 2; 
     private _curAltAGL = ASLToAGL getPosASL _heli # 2;
-    private _curAltASL = getPosASL _heli # 2;
+    if (_curAltAGL > 50) then {
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAltitudeHoldEnable.sqf:8]: Should this here be formatted this way? It isn't for display

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/core/fn_coreUpdate.sqf [hunk 3]
```diff
@@ -24,7 +24,7 @@ if (isGamePaused) exitwith {};
 private _config      = configFile >> "CfgVehicles" >> typeof _heli;
 private _flightModel = getText (_config >> "fza_flightModel");
 
-private _deltaTime   = ((["sfmplus_deltaTime"] call BIS_fnc_deltaTime) min 1/30);
+private _deltaTime   = ["sfmplus_deltaTime"] call BIS_fnc_deltaTime;
```
Comment (mattysmith22) [inline @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/core/fn_coreUpdate.sqf:27]: This looks like an unrelated change?

**Review by mattysmith22 — CHANGES_REQUESTED** [review]

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAltitudeHoldEnable.sqf (hunk 2 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/fmc/fn_fmcAltitudeHoldEnable.sqf:8]: It's there on purpose to ensure that when the Radar Altitude is > 50 ft, instead of trying to maintain a 1 foot altitude resolution, it maintains a 10 foot resolution, which smooths out output to the rotor.

File: @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/core/fn_coreUpdate.sqf (hunk 3 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/core/fn_coreUpdate.sqf:27]: This is the correct implementation of deltaTime to ensure the holds and simulation run smoothly, which was required for testing. It's not unrelated.

File: @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageFLTDraw.sqf (hunk 1 above)
Comment (BradMick) [reply @AH-64D Apache Longbow/Addons/fza_ah64_mpd/functions/page/fn_pageFLTDraw.sqf:27]: Fixed

**Review by mattysmith22 — APPROVED** [review]
